src/index.js

```diff
@@ -1,16 +1,17 @@
 import './style.css';
-import scores from './module/database.js';
+import newCreatedGame from './module/generateId.js';
+import saveScore from './module/saveToDb.js';
+import displayData from './module/fetchData.js';
 
-const container = document.querySelector('.container');
+const submit = document.getElementById('submit');
+submit.addEventListener('click', (e) => {
+  e.preventDefault();
+  saveScore();
+});
 
-const displayScore = () => {
-  container.innerHTML = '';
-  for (let i = 0; i < scores.length; i += 1) {
-    container.innerHTML += `
-    <li class='${scores[i].id % 2 ? 'odd' : 'even'}'>${scores[i].name} : ${scores[i].score}</li>
-    </ul>
-    `;
-  }
-};
+const refresh = document.getElementById('refresh');
+refresh.addEventListener('click', () => {
+  displayData();
+});
```

Comment:
- [ ] @Theodoraldo great work on your app, however, I noticed a small error whenever I click on the refresh button after adding 1 or 2 scores it duplicates the scores on display as demonstrated below and I have to refresh the page to get scores stored in the API.

## First refresh click
![image](https://github.com/Theodoraldo/leaderboard-project/assets/61408860/d6252ef1-82d1-4bc4-978d-3b02859d981b)
## Second refresh click
![image](https://github.com/Theodoraldo/leaderboard-project/assets/61408860/bdbdfb91-46f7-43d1-9ee9-0885046b420d)
## Third refresh click
![image](https://github.com/Theodoraldo/leaderboard-project/assets/61408860/bf9e6d5a-a171-42f5-82ba-c006b63c1241)